coderd/database/migrations/000250_email_reports.up.sql

```diff
@@ -0,0 +1,32 @@
+INSERT INTO notification_templates (id, name, title_template, body_template, "group", actions)
+VALUES ('34a20db2-e9cc-4a93-b0e4-8569699d7a00', 'Report: Workspace Builds Failed For Template', E'Workspace builds failed for template "{{.Labels.template_display_name}}"',
+        E'Hi {{.UserName}},
+
+Template **{{.Labels.template_display_name}}** has failed to build {{.Data.failed_builds}}/{{.Data.total_builds}} times over the last {{.Data.report_frequency}} and may be unstable.
+
+**Report:**
+{{range $version := .Data.template_versions}}
+**{{$version.template_version_name}}** failed {{$version.failed_count}} time{{if gt $version.failed_count 1}}s{{end}}:
+{{range $build := $version.failed_builds}}
+* [{{$build.workspace_owner_username}} / {{$build.workspace_name}} / #{{$build.build_number}}]({{base_url}}/@{{$build.workspace_owner_username}}/{{$build.workspace_name}}/builds/{{$build.build_number}})
```

Comment:
Nit: I think a table display might be nicer here.

Reply:
I'm not aware of any problems with tables in email; could you elaborate? Is it a technical issue, or more of a UX one?

Reply:
UX, this is how I perceive tables used in emails (google some basic samples), especially if you don't know the size of the table

1. You need to scroll the screen:

<img width="500" alt="Screenshot 2024-09-13 at 15 51 24" src="https://github.com/user-attachments/assets/b3badfaa-7160-4b8d-b998-15b69a9c4b00">

2. Squized:

<img width="617" alt="Screenshot 2024-09-13 at 15 51 31" src="https://github.com/user-attachments/assets/054bbddb-2dc8-42af-8fd6-a157be4d1a86">

BTW What do you think is the best table for this use case? A few tables, one per template version? or a single big one? I'm trying to visualize it.